examples/tabular/example_tabular_predictor_v2.py

```diff
@@ -0,0 +1,62 @@
+
+from autogluon.tabular import TabularPrediction as task
+from autogluon.tabular.task.tabular_prediction.predictor import TabularPredictor
+from autogluon.tabular.task.tabular_prediction.predictor_v2 import TabularPredictorV2
+
+
+################
+# Loading data #
+################
+
+train_data = task.Dataset(file_path='https://autogluon.s3.amazonaws.com/datasets/Inc/train.csv')
+test_data = task.Dataset(file_path='https://autogluon.s3.amazonaws.com/datasets/Inc/test.csv')
+label = 'class'
+hyperparameters = {'RF': {}}
+train_data = train_data.head(1000)  # subsample for faster demo
+
+##################################
+# Fitting with the old Predictor #
+##################################
+
+predictor1 = task.fit(
+    train_data=train_data,
+    label=label,
+    hyperparameters=hyperparameters,
+)
+predictor1.leaderboard(test_data)
+
+##################################
+# Fitting with the new Predictor #
+##################################
+
+predictor2 = TabularPredictorV2(label=label)
+
+# Showing that the new Predictor can be saved and loaded prior to calling .fit()
+output_dir = predictor2.output_directory
+predictor2.save()
+del predictor2
+predictor2 = TabularPredictorV2.load(output_directory=output_dir)
+
+predictor2.fit(
+    train_data=train_data,
+    hyperparameters=hyperparameters,
+)
+predictor2.leaderboard(test_data)
+
+# Showing that the new Predictor can be reloaded after fit in the same fashion as the old Predictor
+predictor2.save()
+del predictor2
+predictor2 = TabularPredictorV2.load(output_directory=output_dir)
```

Comment:
similar to above, we can hide `output_directory=`. besides, we may call it `directory`, the reason is that `output_directory` means the  output directory of the previous predictor when we save it, while `load` focus on the current predictor

Reply:
Updated